pandas/core/strings/accessor.py

```diff
@@ -2214,32 +2214,33 @@ def wrap(self, width: int, **kwargs):
         r"""
         Wrap strings in Series/Index at specified line width.
 
-        This method has the same keyword parameters and defaults as
-        :class:`textwrap.TextWrapper`.
-
         Parameters
         ----------
         width : int
             Maximum line width.
-        expand_tabs : bool, optional
```

Comment:
instead could you change the signature of `wrap` to include these parameters?

Reply:
thanks for the suggestion, I've made this update!